.github/workflows/pytest.yml

```diff
@@ -0,0 +1,35 @@
+name: Python package
+
+on: [push]
```

Comment:
Can we have it run on pull and pushes to the master branch?

Reply:
Sure - I've added that 👍